### PR TITLE
fix: [M3-6387] - MUI v5 Migration - Components > SelectionCard & ActionsPanel

### PR DESCRIPTION
**Pull request by jaalah-akamai**
## Description 📝
The parent components for Marketplace selection cards needed to be updated as well with new Gridv2.

## Major Changes 🔄 
- Updated all Grid references that used `SelectionCard`
- Migrated `ActionsPanel` since that was connected to one of the files affected

## Preview 📷
| Before      | After |
| ----------- | ----------- |
|Screen Shot 2023-03-29 at 4 32 55 PM|Screen Shot 2023-03-29 at 4 31 25 PM|

Screen Shot 2023-03-29 at 6 00 39 PM

## How to test 🧪
- Compare http://localhost:3000/linodes/create?type=One-Click with Prod
- Ensure "Make a Payment" is still good: http://localhost:3000/account/billing/make-payment
- User Permissions: http://localhost:3000/account/users/{USER}/permissions

### CODE REVIEW

File: packages/manager/src/components/SelectionCard/SelectionCard.tsx [hunk 1]
```diff
@@ -29,12 +29,6 @@ export interface Props {
 const StyledGrid = styled(Grid, {
   label: 'SelectionCardGrid',
 })<Partial<Props>>(({ theme, ...props }) => ({
-  minWidth: 200,
-  padding: theme.spacing(2),
-  justifyContent: 'center',
-  alignItems: 'center',
-  display: 'flex',
-  outline: 0,
```
Comment (jaalah-akamai) [inline packages/manager/src/components/SelectionCard/SelectionCard.tsx:37]: No need for any of these styles anymore, the Grid should handle all this.

File: packages/manager/src/features/Billing/BillingPanels/BillingSummary/PaymentDrawer/PaymentMethodCard.tsx [hunk 2]
```diff
@@ -83,6 +83,7 @@ export const PaymentMethodCard = (props: Props) => {
 
   return (
     <Grid
+      container
```
Comment (jaalah-akamai) [inline packages/manager/src/features/Billing/BillingPanels/BillingSummary/PaymentDrawer/PaymentMethodCard.tsx:86]: This was the main fix

File: packages/manager/src/features/linodes/LinodesCreate/AppPanelSection.tsx [hunk 3]
```diff
@@ -2,24 +2,16 @@ import { StackScript, UserDefinedField } from '@linode/api-v4/lib/stackscripts';
...
-const useStyles = makeStyles((theme: Theme) => ({
-  flatImagePanelSelections: {
-    marginTop: theme.spacing(2),
-    marginBottom: theme.spacing(),
-    padding: `${theme.spacing(1)} 0`,
-  },
-  chip: {
-    '& span': {
-      color: 'inherit !important',
-    },
```
Comment (jaalah-akamai) [inline packages/manager/src/features/linodes/LinodesCreate/AppPanelSection.tsx:21]: This style wasn't doing much at all, so I removed.

File: packages/manager/src/features/linodes/LinodesCreate/SelectionCardWrapper.tsx [hunk 4]
```diff
@@ -1,34 +1,11 @@
...
 import Grid from 'src/components/Grid';
 import SelectionCard from 'src/components/SelectionCard';
 import { APP_ROOT } from 'src/constants';
 
-const useStyles = makeStyles((theme: Theme) => ({
-  flatImagePanelSelections: {
-    marginTop: theme.spacing(2),
-    marginBottom: theme.spacing(),
-    padding: `${theme.spacing(1)} 0`,
-  },
```
Comment (jaalah-akamai) [inline packages/manager/src/features/linodes/LinodesCreate/SelectionCardWrapper.tsx:15]: `flatImagePanelSelections` here was unused

**Review by hana-linode — CHANGES_REQUESTED** [review]
Looks like the layout is wonky in some other places that use SelectionCard

image

Linode Plan mobile view
image

**Comment by jaalah-akamai** [comment]
@hana-linode should be working again!

**Review by hana-linode — CHANGES_REQUESTED** [review]
Just noticed some minor regressions in the Make a Payment drawer, but other than that I think we're good!

This branch:
Chip text being cut off on mobile screens and extra spacing between Pay Now button
image

Prod:
image

**Comment by jaalah-akamai** [comment]
The space between the "Pay Now" button will be addressed in [PR #8933](https://github.com/linode/manager/pull/8933/files#diff-8a21e23fcac02097f86824bdd156bc44d90851c4c159b1503c66cf78316ebaca) with changes to `PaymentDrawer.tsx` where I update the Grid for that file. I'll make sure to check this on that end 👍

**Review by hana-linode — APPROVED** [review]

**Pull request merged**